scapy/utils.py

```diff
@@ -75,7 +75,7 @@ def lhex(x):
 
 def to_hex(x):
     """Turns string or bytes to hex format"""
-    return x if isinstance(x, bytes) and not six.PY2 else x.encode("hex")
+    return bytes_hex(x).decode()
```

Comment:
What is the expected difference between `to_hex()` and `bytes_hex()`?

Reply:
to_hex should return hex as string, whereas bytes_hex returns hex as bytes.

Reply:
What do you think of using explicitly `bytes_hex(x).decode()` and remove `to_hex()`? I find this a bit confusing.